lib/bolt/node.rb

```diff
@@ -10,7 +10,7 @@ class Node
     ENVIRONMENT_METHODS = %w[both environment].freeze
 
     def self.from_uri(uri_string, **kwargs)
-      uri = NodeURI.new(uri_string)
+      uri = NodeURI.new(uri_string, kwargs[:config][:transport])
```

Comment:
I guess `uri_string` here could already include the transport like `winrm://`, so you couldn't just concat like this and leave `node_uri.rb` alone?

```ruby
NodeURI.new("#{kwargs[:config][:transport]}://uri_string)
```

Reply:
right, it's the default transport, only used if the `uri_string` doesn't specify a scheme.